ui/src/router/index.js

```diff
@@ -44,6 +44,14 @@ const routes = [
     // which is lazy-loaded when the route is visited.
     component: () => import(/* webpackChunkName: "modules" */ '../views/Modules.vue')
   },
+  {
+    path: '/swagger',
+    name: 'swagger',
+    // route level code-splitting
+    // this generates a separate chunk (modules.[hash].js) for this route
+    // which is lazy-loaded when the route is visited.
+    component: () => import(/* webpackChunkName: "modules" */ '../views/Swagger.vue')
+  },
```

Comment:
This is just copied from similar routes here and it works so I didn't touch it much more